### PR TITLE
feat: adding textarea disabled style

### PR DESCRIPTION
**Pull request by palbizu**
## Description
Adding textarea disabled style.

### Testing
Visual testing

image

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/components/src/textarea/textarea.component.scss [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 :host {
   ::ng-deep {
+    &.disabled {
```
Comment (aaron-steinfeld) [inline projects/components/src/textarea/textarea.component.scss:5]: Not needed inside ng deep, we're adding this class

Comment (palbizu) [reply projects/components/src/textarea/textarea.component.scss:5]: it's necessary to keep it to apply the styles

Comment (aaron-steinfeld) [reply projects/components/src/textarea/textarea.component.scss:5]: What am I missing? It looks like it's just a regular style on an element of our own template.

Comment (palbizu) [reply projects/components/src/textarea/textarea.component.scss]: My bad, I made a mistake when I tested it. Styles moved.

File: projects/components/src/textarea/textarea.component.scss [hunk 2]
```diff
@@ -1,6 +1,12 @@
 @import 'mixins';
 
 :host {
+  .disabled {
```
[the remark this answers is not in the document]
Comment (palbizu) [reply projects/components/src/textarea/textarea.component.scss]: done

**Pull request merged**